core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/InfoSubcommand.java

```diff
@@ -54,7 +55,7 @@ public void execute(@NotNull CommandSource sender, @NotNull String label, @NotNu
                 placeholder("version", main.getVersion()),
                 placeholder("description", main.getDescription()),
                 placeholder("credits", "https://github.com/lokka30/Treasury/wiki/Credits"),
-                placeholder("latest-api-version", main.getEconomyAPIVersion()),
+                placeholder("latest-api-version", EconomyAPIVersion.getLatestVersion()),
```

Comment:
Same comment here as above, I think this should be renamed to `current-api-version` for the same reason